benchmark.py

```diff
@@ -78,7 +78,7 @@ def main(job_config: JobConfig):
         batch_size=job_config.training.batch_size,
         extra_args=[],
     )
-    model_flops = get_model_flops(config)
+    #model_flops = get_model_flops(config)
```

Comment:
```suggestion
    # model_flops = get_model_flops(config)
```
also comment on why removing it -- e.g. it is Llama-specific code

Reply:
This is for benchmark models. `get_model_flops` is called before compilation. Some of the benchmarked model will OOM, and this function is only for printing model information. So I just commented it.